eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -157,14 +166,25 @@ private void checkParameters(String variantCoreString, String annotationVepVersi
     }
 
     @GetMapping(value = "/{variantCoreString}/annotations")
-    public ResponseEntity getAnnotations(@PathVariable("variantCoreString") String variantCoreString,
-                                         @RequestParam(name = "species") String species,
-                                         @RequestParam(name = "assembly") String assembly,
-                                         @RequestParam(name = "annot-vep-version", required = false)
-                                                 String annotationVepVersion,
-                                         @RequestParam(name = "annot-vep-cache-version", required = false)
-                                                 String annotationVepCacheVersion,
-                                         HttpServletResponse response) throws IllegalArgumentException {
+    public ResponseEntity getAnnotations(
+            @ApiParam(value = "Internal species-relative variant ID. Chromosome, start, reference allele and" +
```

Comment:
I'm not sure the first part is very informative.
```suggestion
            @ApiParam(value = "Chromosome, start, reference allele and" +
```